processor/geoipprocessor/go.mod

```diff
@@ -14,6 +14,11 @@ require (
 	go.uber.org/goleak v1.3.0
 )
 
+require (
+	github.com/maxmind/mmdbwriter v1.0.0 // indirect
+	go4.org/netipx v0.0.0-20230824141953-6213f710f925 // indirect
+)
```

Comment:
why is this a separate set

Reply:
not sure, it was automatically added by my editor's formatter (maybe when working from a go.mod subdirectory?). Require blocks unified in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33451/commits/05458bf369a40e0d90dbd05c502bdba879cf3333